wiki/Contests/Official_support/en.md

```diff
@@ -72,6 +73,19 @@ If approved, you will need to submit the following:
 
 In exceptionally rare cases, a contest may be granted the privilege of awarding badges to its top three finalists. A contest that has received this distinction is recognised for attaining the highest level of production quality and organisation possible — this is given exclusively at the discretion of the osu! team. Hosts will be contacted directly if applicable.
 
+### News posts
+
+While support for advertising and sharing the outcome of your contest is available via news posts, there are additional requirements that must be considered:
+
+- Contests will be expected to provide the following news posts:
+  - A news post for advertising the contest.
+  - A news post announcing that the voting is open (if the winners are determined based on public voting).
+  - A news post detailing **final** results.
+Those responsible for writing them should expect to adhere to the editorial standards required by our [news styling criteria](/wiki/News_styling_criteria). Additional news posts are presently not permitted.
+- If your tournament is limited to a specific region, it is only eligible to receive news post support if it involves a LAN (a.k.a. an in-person meetup).
```

Comment:
although i'm not sure how relevant this rule is

```suggestion
- If your contest is limited to a specific region, it is only eligible to receive news post support if it involves a LAN (a.k.a. an in-person meetup).
```

Reply:
dont think its very relevant and it looks i left it in my copy pasta by accident, lets see what the committee says

Reply:
https://discord.com/channels/188630481301012481/218677502141399041/1246861691061538887

> RandomeLoL — Today at 18:23
> The clause is fine for us. They're relevant in the off-chance one is done

Reply:
Relevant for contests in the off-chance one is held off in an in-person meetup. We'd rather have that covered before needing to make an exception later down the road.